docs/self-hosted/install/install-on-kubernetes.md

```diff
@@ -24,18 +24,26 @@ For some cloud providers, we offer [Terraform](https://www.terraform.io/) script
 
 To install Gitpod in your Kubernetes cluster, follow these steps:
 
+1. Create a file `values.minio-secrets.yaml` with the following content:
```

Comment:
Later we already create a `values.custom.yaml` where we put all configuration values (those are not too many). Would be nice to re-use it for this as well.